tests/bluetooth/common/testlib/src/sync.c

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/sys/util.h>
+#include "argparse.h"
+#include "bs_types.h"
+#include "bs_tracing.h"
+#include "time_machine.h"
+#include "bs_pc_backchannel.h"
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(sync, CONFIG_LOG_DEFAULT_LEVEL);
+
+#define CHANNEL_ID 0
+#define MSG_SIZE 1
+
+int bs_sync_init(void)
+{
+	uint device_number = get_device_nbr();
+	uint peer_number = device_number ^ 1;
```

Comment:
You'd need to mention in the doxygen comment that this will only work if there is 2 devices in the test with device numbers 0 and 1

Reply:
May be worth checking if the backchannel functions we use in LE Audio (designed to work with an arbitrary number of devices) may be useful to define here: https://github.com/zephyrproject-rtos/zephyr/blob/main/tests/bsim/bluetooth/audio/src/common.c#L170-L329